src/routes/recipes/+page.svelte

```diff
@@ -152,6 +152,21 @@
 		top: 4px;
 	}
 
+	.list-title::before {
+		content: '';
+		background: var(--black);
+		opacity: 0.8;
+		height: 0px;
+		width: 100%;
+		position: absolute;
+		bottom: 5px;
+		transition: all 0.05s ease;
```

Comment:
```suggestion
		transition: height 0.05s ease;
```
The property that you want to animate is `height` not all possible properties.
To avoid side effect, it's better to target only the thing that you are changing.